### ISSUE TITLE
Enum values start with `1`

### ISSUE THREAD

**Issue by Bhacaz** [petalmd/rubocop-petal#56]
```
https://github.com/rails/rails/issues/47868
https://github.com/rails/rails/issues/41595
https://github.com/rails/rails/pull/41599
```

The idea to start with 1 is that

When using a wrong value like `where(state: :patato)` let Rails + MySQL do a `WHERE state = 0`. It will match nothing since no record will have a 0 value.

```ruby
# bad
enum state: { open: 0, close: 1 }

# good
enum state: { open: 1, close: 2}
```

**Comment by faridanoosheh** [comment]
here is a new Cop https://github.com/petalmd/rubocop-petal/pull/57 to be review

**Comment by Bhacaz** [comment]
Fix in Rails 7.0
